File: src/main/java/dto/PackageDTO.java
package dto;

import java.time.LocalDate;
import java.util.List;

import entity.Cost;
import entity.Departure;
import entity.Itinerary;
import entity.Booking;
import lombok.*;
import entity.Subcategory;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PackageDTO {
    private Integer packageId;
    private String packageName;
    private String packageInfo;
    private String packageImagePath;
    private Integer durationDays;
    private LocalDate startDate;
    private LocalDate endDate;

    private Integer subcategoryId;

//    private List<Itinerary> itineraries;
    private List<ItineraryDTO> itineraries; 


    private List<CostDTO> costs;
    private List<DepartureDTO> departures;
    
}

File: DotNet_Backend/SafarnamaApplication/SafarnamaApplication/DTOs/PackageDTO.cs

using SafarnamaApplication.DTOs;

namespace SafarnamaApplication.DTOs
{
    public class PackageDTO
    {
        public int PackageId { get; set; }
        public int? SubcategoryId { get; set; }
        public int? CategoryId { get; set; }

        public string? PackageName { get; set; }
        public string? PackageInfo { get; set; }
        public string? PackageImagePath { get; set; }
        public int? DurationDays { get; set; }
        public DateOnly? StartDate { get; set; }
        public DateOnly? EndDate { get; set; }

        public string? CategoryName { get; set; }
        public string? SubcategoryName { get; set; }

        public List<CostDTO> Costs { get; set; } = new();
        public List<ItineraryDTO> Itineraries { get; set; } = new();
        public List<DepartureDTO> Departures { get; set; } = new();


    }
}
